src/bodies.cpp

```diff
@@ -759,9 +781,9 @@ void bodies::ConvexMesh::useDimensions(const shapes::Shape *shape)
   mesh_data_->bounding_cylinder_.radius = maxdist;
   mesh_data_->bounding_cylinder_.length = cyl_length;
 
-  FILE* null = fopen ("/dev/null","w");
+  static FILE* null = fopen ("/dev/null","w");
```

Comment:
This is ok, but I believe it is unsafe before c++11 -- there is no guarantee that the first construction is not reentrant.